tests/cli/deliver/test_run_deliver_cmd.py

```diff
@@ -0,0 +1,156 @@
+"""Tests for running the delivery command"""
+
+import logging
+
+from pathlib import Path
+
+from click.testing import CliRunner
```

Comment:
You could reuse the  cli_runner fixture already in base context also! Since you're instantiating the class anyway with every test

Reply:
Please let me skip that